samples/toolbar/toolbar.cpp

```diff
@@ -307,7 +307,8 @@ bool MyApp::OnInit()
     // Create the main frame window
     MyFrame* frame = new MyFrame((wxFrame *) NULL, wxID_ANY,
                                  "wxToolBar Sample",
-                                  wxPoint(100, 100), wxSize(650, 350));
+                                  wxPoint(100, 100), wxDefaultSize);
+    frame->SetSize(frame->FromDIP(wxSize(650, 350)));
```

Comment:
Just a note (i.e. this is not related to this PR per se):

Seeing it like this, it does look ugly. And I'm not even sure if it's identical to the old code, we'd need to use `SetInitialSize()` to really have the same effect.

I think we may need to add a new `wxFrame` ctor allowing to specify the size in DIPs.